install/firmware.csv

```diff
@@ -1734,17 +1706,11 @@ Trimble Navigation Ltd.,TRIMBLE NETRS,4632120756,1.1-5,2006-10-11T00:00:00Z,2008
 Trimble Navigation Ltd.,TRIMBLE NETRS,4632120756,1.2-0,2008-12-05T14:54:46Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120781,1.1-5,2006-10-11T00:00:00Z,2009-04-07T15:01:45Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120781,1.2-0,2009-04-07T15:01:46Z,9999-01-01T00:00:00Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2006-10-11T00:00:00Z,2008-12-10T14:00:16Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-10T14:00:17Z,2008-12-10T14:41:49Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-10T14:41:50Z,2008-12-11T12:01:24Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-11T12:01:25Z,2009-04-07T18:13:14Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.2-0,2009-04-07T18:13:15Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2006-10-11T00:00:00Z,2009-04-07T06:59:29Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.2-0,2009-04-07T06:59:30Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120794,1.1-5,2006-10-11T00:00:00Z,2009-04-08T10:23:06Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120794,1.2-0,2009-04-08T10:23:07Z,9999-01-01T00:00:00Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120795,1.1-5,2006-10-11T00:00:00Z,2008-12-10T16:04:39Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120795,1.1-5,2008-12-10T16:04:40Z,2009-04-08T10:29:56Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120795,1.1-5,2009-04-08T10:29:57Z,2009-04-08T10:47:30Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120795,1.1-5,2009-04-08T10:47:31Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4635120795,1.1-5,2006-10-11T00:00:00Z,9999-01-01T00:00:00Z,
```

Comment:
this one looks good, is a single firmware version that remained the same. Receiver was installed at two different sites, and some randomly sampled days are all showing firmware 1.1-5 was installed, so no need to change this updated entry (2006 to 9999 for 1.1-5)